modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx

```diff
@@ -91,6 +92,276 @@ for alpha components and Preview as a space for beta components.
 We've promoted several components from Labs to Preview in v5. See
 [Component Promotions](#component-promotions) for more details.
 
+### Type Deprecations and Hierarchy Updates
+
+Canvas Kit v4 supported two type hierarchies, `beta` and `legacy` in
+`@workday/canvas-kit-labs-react-core` and `@workday/canvas-kit-react-core` respectively. However, v5
+replaces those with a new, responsive type hierarchy in `@workday/canvas-kit-react-core`. We are
+also deprecating and updating our type variants. The v5 codemod handles almost all of these changes
+for you. That said, you'll want to review the transformation and your UI to ensure everything was
+updated as you expect.
+
+#### Automatic Updates
+
+- 🤖 Type Hierarchy Updates
+
+All type hierarchy updates are handled by the codemod. The tables below will help you understand the
+changes and provide a reference as you review your UI. Most teams are using the `beta` type tokens
```

Comment:
These long blocks of content nested within list items will be set apart a bit nicer if you indent them one level. See [the Automatic Updates section under Core](https://github.com/Workday/canvas-kit/blame/prerelease/v5/modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx#L162) for example. Note how the content in each line of the list item is flush with the 🤖.

Reply:
Ah, I missed that nuance when I was looking at your formatting. Thanks!